### PR TITLE
Bug fix to handle final word in word level argmax

### PR DESCRIPTION
**Pull request by lettergram**

### CODE REVIEW

File: dataprofiler/labelers/data_processing.py [hunk 1]
```diff
@@ -825,8 +825,8 @@ def _word_level_argmax(self, data, predictions, label_mapping,
             label_count = {label_mapping[default_label]: 0}
             for idx in range(len(sample)):
 
-                # Split on separator
-                if sample[idx] in separator_dict:
+                # Split on separator or last sample
+                if sample[idx] in separator_dict or idx == len(sample)-1:
```
Comment (AnhTruong) [inline dataprofiler/labelers/data_processing.py:829]: should we add a test for this?

Comment (lettergram) [reply dataprofiler/labelers/data_processing.py:829]: Very good point, I'll add a test to evaluate this case

File: dataprofiler/labelers/data_processing.py [hunk 2]
```diff
@@ -825,8 +825,8 @@ def _word_level_argmax(self, data, predictions, label_mapping,
             label_count = {label_mapping[default_label]: 0}
             for idx in range(len(sample)):
```
Comment (cbbruss) [inline dataprofiler/labelers/data_processing.py:826]: Would an `enumerate` not be cleaner?

**Review by JGSweets — APPROVED** [review]

**Review by ksneab7 — APPROVED** [review]

File: dataprofiler/labelers/data_processing.py (hunk 2 above)
Comment (JGSweets) [reply dataprofiler/labelers/data_processing.py:824]: agreed, but not going to block the PR if @lettergram doesn't get to it prior to merge

**Pull request merged**